webplugin/js/app/km-widget-custom-events.js

```diff
@@ -215,4 +215,20 @@ var eventMapping = {
         },
         eventFunction: null,
     },
+    onHelpFulClick: {
```

Comment:
do we need 2 separate events? or we can have just one event, and receive the value on that event with 0 or 1
in future, if we add more capabilities say: Great Answer, Loved it, etc then this code will not be scalable